features/machine/machine_healthcheck.feature

```diff
@@ -260,3 +260,18 @@ Feature: MachineHealthCheck Test Scenarios
     Then the output should contain:
       | mhc-<%= machine_set.name %>: total targets: 1,  maxUnhealthy: 1%, unhealthy: 1. Short-circuiting remediation |
     """
+
+  # @author miyadav@redhat.com
+  # @case_id OCP-33714
+  @admin
+  Scenario: Leverage OpenAPI validation within MHC
+    Given I have an IPI deployment
+    And I switch to cluster admin pseudo user
+    Then I use the "openshift-machine-api" project
+
+    # Create MHC with malformed unhealthy nodes value and empty selectors
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/cloud/mhc/mhc_malformed.yaml | 
```

Comment:
```
Given I obtain test data file "cloud/mhc/mhc_malformed.yaml"
When I run the :create client command with:
    | f | mhc_malformed.yaml | 
```

Reply:
just wanted to have less lines of code , I think both ways are ok ...

Reply:
If we some day decide to store test files in some other repo, we can change in one place  instead of changing it everywhere.
```
Given /^I obtain test data (file|dir) #{QUOTED}$/ do |type, path|
  accepted_roots = [
    "#{BushSlicer::HOME}/testdata",
    "#{BushSlicer::HOME}/features/tierN/testdata",
  ]
```

Reply:
done